### PR TITLE
Javascript sample PR

### PR DESCRIPTION
**Pull request by dodaromike**

### CODE REVIEW

File: graph-tutorial/index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    
+      </main>
+    
+      <!-- Bootstrap -->
+      <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+              integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
+              crossorigin="anonymous"></script>
+    
+      <!-- Moment.js -->
+      <script src="https://cdn.jsdelivr.net/npm/moment@2.29.1/moment.min.js"></script>
```
Comment (jasonjoh) [inline graph-tutorial/index.html:51]: Update moment.js

```suggestion
      <script src="https://cdn.jsdelivr.net/npm/moment@2.29.4/moment.min.js"></script>
```

File: graph-tutorial/index.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      <script src="https://momentjs.com/downloads/moment-timezone-with-data-10-year-range.js"></script>
+    
+      <!-- MSAL -->
+      <script src="https://alcdn.msauth.net/browser/2.16.1/js/msal-browser.min.js"
+              integrity="sha384-bPBovDNeUf0pJstTMwF5tqVhjDS5DZPtI1qFzQI9ooDIAnK8ZCYox9HowDsKvz4i"
+              crossorigin="anonymous"></script>
```
Comment (jasonjoh) [inline graph-tutorial/index.html:57]: Please update

```suggestion
      <script src="https://alcdn.msauth.net/browser/2.28.1/js/msal-browser.min.js"
              integrity="sha384-ei8xVSyFPTuRnbO1sdYy5qJT6Kd9neBfVG8AjZySEwdMG1GhCThbceSqxJnx0Ci3"
              crossorigin="anonymous"></script>
```

File: graph-tutorial/index.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    
+      <!-- Graph SDK -->
+      <script src="https://cdn.jsdelivr.net/npm/@microsoft/microsoft-graph-client@3.0.0/lib/graph-js-sdk.js"></script>
+      <script src="https://cdn.jsdelivr.net/npm/@microsoft/microsoft-graph-client@3.0.0/lib/graph-client-msalBrowserAuthProvider.js"></script>
```
Comment (jasonjoh) [inline graph-tutorial/index.html:61]: Please update

```suggestion
      <script src="https://cdn.jsdelivr.net/npm/@microsoft/microsoft-graph-client@3.0.2/lib/graph-js-sdk.js"></script>
      <script src="https://cdn.jsdelivr.net/npm/@microsoft/microsoft-graph-client@3.0.2/lib/graph-client-msalBrowserAuthProvider.js"></script>
```

**Review by jasonjoh — CHANGES_REQUESTED** [review]
Please update the dependencies in index.html and test that it all still works. Also, we're missing config.example.js.

File: README.md [hunk 4]
```diff
@@ -1,12 +1,63 @@
...
-The code for this sample is in the [demo](demo) folder. Instructions to configure and run the sample can be found in the [README](demo/README.md) in that folder.
+1. Select **New registration**. On the **Register an application** page, set the values as follows.
+
+    - Set **Name** to `JavaScript Graph Tutorial`.
+    - Set **Supported account types** to **Accounts in any organizational directory and personal Microsoft accounts**.
+    - Under **Redirect URI**, set the first drop-down to `Single-page application (SPA)` and set the value to `http://localhost:8080`.
+
+1. Choose **Register**. On the **JavaScript Graph Tutorial** page, copy the value of the **Application (client) ID** and save it, you will need it in the next step.
+
+## Configure the sample
```
[the remark this answers is not in the document]
Comment (dodaromike) [reply README.md:44]: Took out versions in microsoft-graph-client and npm/moment dependencies.  Without bootstrap@5.0.2 there are display problems.

File: graph-tutorial/graph.js [hunk 5]
```diff
@@ -77,6 +77,72 @@ async function getEvents() {
 }
 // </getEventsSnippet>
 
+// <showCalendarSnippet>
+function showCalendar(events) {
```
[the remark this answers is not in the document]
Comment (dodaromike) [reply graph-tutorial/graph.js:81]: I committed some of your changes, and now I can't use any git commands from my local branch.  There is some kind of ownership problem:
C:\Users\v-dodaromike\ConvertSamples\JavaScriptSPA\msgraph-training-javascriptspa>git status
fatal: detected dubious ownership in repository at 'C:/Users/v-dodaromike/ConvertSamples/JavaScriptSPA/msgraph-training-javascriptspa'
To add an exception for this directory, call:

        git config --global --add safe.directory C:/Users/v-dodaromike/ConvertSamples/JavaScriptSPA/msgraph-training-javascriptspa

Set the environment variable GIT_TEST_DEBUG_UNSAFE_DIRECTORIES=true and run
again for more information.

**Review by jasonjoh — APPROVED** [review]

**Pull request merged**